src/PublishFilePanel.cpp

```diff
@@ -18,11 +18,11 @@ PublishFilePanel::PublishFilePanel(BMessenger* target)
 		{
 			BView* poseView = v->FindView("DirMenuField");
 			BView* parentview;
-			BEntry rst2pdfcheck("/boot/common/bin/rst2pdf");
-			BEntry rst2htmcheck("/boot/common/bin/rst2html.py");
-			BEntry rst2xmlcheck("/boot/common/bin/rst2xml.py");
-			BEntry rst2odtcheck("/boot/common/bin/rst2odt.py");
-			BEntry rst2texcheck("/boot/common/bin/rst2latex.py");
+			BEntry rst2pdfcheck("/boot/system/bin/rst2pdf-3.10");
```

Comment:
This will break after python update

Reply:
Yeah, was thinking about it, maybe just keep it to the default one.

Reply:
Suffix less was introduced here: https://github.com/haikuports/haikuports/pull/8134
So these always fall back to the "default" python version in use there.